staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -240,7 +240,8 @@ func (s *DelegatingAuthenticationOptions) getClientCA() (*ClientCertAuthenticati
 		return nil, err
 	}
 	if incluster == nil {
```

Comment:
we should have an option like `SkipInClusterClientCALookup` or something, similarly to `SkipInClusterLookup`, or perhaps change the behavior of `SkipInClusterLookup`

@kubernetes/sig-api-machinery-pr-reviews WDYT?

Reply:
or maybe @kubernetes/sig-auth-pr-reviews